projects/plugins/jetpack/extensions/shared/external-media/index.js

```diff
@@ -17,6 +20,15 @@ function insertExternalMediaBlocks( settings, name ) {
 }
 
 if ( isCurrentUserConnected() && 'function' === typeof useBlockEditContext ) {
+	const isInserterOpened = () =>
+		select( 'core/edit-post' )?.isInserterOpened() ||
+		select( 'core/edit-site' )?.isInserterOpened() ||
+		select( 'core/edit-widgets' )?.isInserterOpened?.();
+
+	waitFor( isInserterOpened ).then( () =>
+		dispatch( 'core/block-editor' )?.registerInserterMediaCategory( getPexelsMediaCategory() )
```

Comment:
This fails for cases where `registerInserterMediaCategory` is unavailable (e.g. WP 6.2 without the Gutenberg plugin active).

<img width="855" alt="Screenshot 2023-07-24 at 17 02 39" src="https://github.com/Automattic/jetpack/assets/1233880/0df7d285-6687-477f-9ab9-e3a66601a967">



```suggestion
		dispatch( 'core/block-editor' )?.registerInserterMediaCategory?.( getPexelsMediaCategory() )
```